testdata/travis.bash

```diff
@@ -1,6 +1,26 @@
+#!/bin/bash
 set -ex
+
+# workaround for go1.10's no module support, we copy this run's source code into namecoin/certinject
```

Comment:
Can you elaborate this comment a bit?  Not totally clear to me at first glance why `$TRAVIS_REPO_SLUG` is relevant.